make/autoconf/build-aux/autoconf-config.guess

```diff
@@ -1270,6 +1270,7 @@ EOF
 		      UNAME_PROCESSOR="x86_64"
 		  fi
 		fi ;;
+	    arm) UNAME_PROCESSOR=aarch64 ;;
```

Comment:
This was not needed for jdk17+: why is it needed here?

Reply:
I had to add it because the `configure` in JDK 11 detects the target processor as 32-bit ARM instead of AArch64.
I don't see the problem with JDK 17.

Reply:
I don't know what JDK 17's `configure` does for detecting the target processor on macOS.

Reply:
OK, I changed my commit to detect the target processor on macOS in the same way as JDK 17 does, adding some lines to make/autoconf/build-aux/config.guess instead of autoconf-config.guess.

Reply:
That change to `config.guess` appears to have originated from Azul, and so, I think we should also include the copyright addition from https://github.com/ibmruntimes/openj9-openjdk-jdk17/commit/e5ed92f29af6833890c1d511b5120e7e71bbf0c5#diff-93f22bc9c1a02cab4846b6c121a1ef46b9451060ec5fb84f7e872a2b8c23dce4.

Reply:
I added the Azul copyright line to config.guess.